.agents/skills/career-ops/SKILL.md

```
@@ -0,0 +1,98 @@
+
+Inbox: add URLs to data/pipeline.md → /career-ops pipeline
+Or paste a JD directly to run the full pipeline.
+```
+
+---
+
+## Context Loading by Mode
+
+After determining the mode, load the necessary files before executing:
+
+### Modes that require `_shared.md` + their mode file:
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)